pkg/adapter/v2/main.go

```diff
@@ -114,6 +114,16 @@ func MainWithContext(ctx context.Context, component string, ector EnvConfigConst
 	adapter := ctor(ctx, env, eventsClient)
 
 	// Build the leader elector
+	leConfig, err := env.GetLeaderElectionConfig()
```

Comment:
why do we want leader election on receive adapters?

Reply:
to reduce downtime when upgrading knative or a worker node.